README.md

```diff
@@ -1467,3 +1476,17 @@ The script serves the Swagger editor locally at a URL where your current version
 of the API documentation is visible. Change the docs in `azul/service/app.py`,
 save, and refresh the page and your changes will appear immediately along with
 any warnings or errors they may introduce.
+
+# 13. Onboarding
+ To assist with getting onboarded to the project, review the following documents:
+
+- [Azul Overview](#1-architecture-overview)  
+- [Getting Started](#2-getting-started)
+- [Contributing Guidelines](CONTRIBUTING.rst)
+  
+Request Access to Cloud Infrastructure and Tools:
```

Comment:
For future reference, we use vertical space of one line between paragraphs in this document. A bullet item is a paragraph by our definition.